convert_hf_to_gguf.py

```diff
@@ -2702,6 +2703,78 @@ class StarCoder2Model(Model):
     model_arch = gguf.MODEL_ARCH.STARCODER2
 
 
+@Model.register("Rwkv6ForCausalLM")
+class RwkvModel(Model):
+    model_arch = gguf.MODEL_ARCH.RWKV
```

Comment:
This is a good point. When writing this part, I only thought about v6 support and possibly v5 (since it's pretty easy to keep the same arch if it's just for these two versions).
I guess it's better using MODEL_ARCH.RWKV6 here. Thanks!